fastlane/Fastfile

```diff
@@ -564,24 +617,24 @@ end
   # Example:
   # bundle exec fastlane build_and_upload_installable_build
   #####################################################################################
-  desc "Builds and uploads an installable build"
-  lane :build_and_upload_installable_build do | options |
+  desc 'Builds and uploads an installable build'
+  lane :build_and_upload_installable_build do |_options|
```

Comment:
If `|_options|` is not used, we might as well remove the unused block param completely.
```suggestion
  lane :build_and_upload_installable_build do
```